modules/drivers/bigquery-cloud-sdk/src/metabase/driver/bigquery_cloud_sdk.clj

```diff
@@ -52,9 +52,9 @@
   '("https://www.googleapis.com/auth/bigquery"
     "https://www.googleapis.com/auth/drive"))
 
-(defn- database->client
-  ^BigQuery [database]
-  (let [creds   (bigquery.common/database-details->service-account-credential (:details database))
+(defn- database-details->client
```

Comment:
I changed this to take database details instead of the database because it was being called in the `driver/can-connect?` implementation inappropriately, by calling it with `{:details details}` instead of an actual database.